atlasdb-client/src/main/java/com/palantir/atlasdb/util/AtlasDbMetrics.java

```diff
@@ -79,4 +84,28 @@ public static void registerCache(MetricRegistry metricRegistry, Cache<?, ?> cach
                     metricsPrefix, existingMetrics);
         }
     }
+
+    private static <T, U extends T> T instrument(
+            MetricRegistry metricRegistry,
+            Class<T> serviceInterface,
+            U service,
+            String name,
+            InstrumentationFilter instrumentationFilter) {
+        return Instrumentation.builder(serviceInterface, service)
+                .withFilter(instrumentationFilter)
```

Comment:
This should be the only change here, the diff is weird because I'm making this into a private method.